linux-bore/PKGBUILD

```diff
@@ -156,6 +156,9 @@ _use_kcfi=${_use_kcfi-}
 # If you use ZFS, refrain from building the RT kernel
 _build_zfs=${_build_zfs-}
 
+# Build the nvidia module in to the kernel
+_build_nvidia=${_build_nvidia-}
+
```

Comment:
I would add here a explanation, what this is exactly doing.
Something like:

```
# Builds the nvidia module and package it into a own base
# This does replace the requirement of nvidia-dkms
```

or something equal

Reply:
Agreed. Will revise once testing is complete.

Reply:
Done.